server/nodejs/test/basicRoutingTest.js

```diff
@@ -0,0 +1,22 @@
+const app = require('../src/app');
+const chai = require('chai');
+const should = chai.should();
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+
+
+
+describe("Simple routing testing", function() {
+    it("/ should return empty JSON with 200 OK", function(done)
+    {
+        chai.request(app)
+        .get('/')
+        .end((err, res) => {
```

Comment:
Don't know if `chai` supports it (I guess it is), but I think using `async/await` is more readable and would make it easier to understand your code